pkgs/build-support/fetchgit/nix-prefetch-git

```diff
@@ -288,7 +297,7 @@ _clone_user_rev() {
         echo "removing \`.git'..." >&2
         find "$dir" -name .git\* -print0 | xargs -0 rm -rf
     else
-        find "$dir" -name .git | while read gitdir; do
+        find "$dir" -name .git | while read -r gitdir; do
```

Comment:
I don't think any result from this command would ever end with a trailing backslash.